### PR TITLE
docs(en): add hosting on 21YunBox

### PR DESCRIPTION
**Pull request by tobyglei**
Create a new PR here.

I submitted a PR at https://github.com/gohugoio/hugo/pull/8121/commits/74d069806b23f7223404204d2637f26c0b04ba5f
weeks ago and looked like nobody review there.

### CODE REVIEW

**Comment by tobyglei** [comment]
@bep can anyone review this? 

21YunBox is the solution for getting hugo to work in China. Hope this deployment guide help making deployment hassle-free in China..

**Comment by bep** [comment]
@anthonyfok 21YunBox currently gives me 1750 hits when searching on Google (which may be a language thing); do you think this is "notable enough"?

**Comment by tobyglei** [comment]
> @anthonyfok 21YunBox currently gives me 1750 hits when searching on Google (which may be a language thing); do you think this is "notable enough"?

How about you search "21云盒子" on Google, it's the 21YunBox in Chinese.

I would say it's the language thing😉 that why you got limited search results.

image

**Comment by anthonyfok** [comment]
@tobyglei wrote:
> I submitted a PR at gohugoio/hugo@74d0698

It is better to reference the pull request at gohugoio/hugo#8121 rather than the commit hash.  ;-)
But yes, since this belongs to general documentation and not tied to a specific feature or version of Hugo,
here at `hugoDocs` is much more appropriate place to discuss this PR, hence I have closed your old PR over at the `hugo` repository.  (More comments to come later.)

**Comment by anthonyfok** [comment]
@bep wrote:
> 21YunBox currently gives me 1750 hits when searching on Google (which may be a language thing); do you think this is "notable enough"?

@tobyglei replied:
> How about you search "21云盒子" on Google, it's the 21YunBox in Chinese.
> I would say it's the language thing:wink: that why you got limited search results.
> *(screenshot with 32,000,000 results)*

But not if the search term is double-quoted like so: `"21云盒子"`, which exclude results like "21", "盒子", and we are left with "about 2,040 results", so, no, unfortunately, it is not a "language thing" in this particular case.

Baidu does give 198,000 results though, so perhaps it is more notable there.

From what I can gather thus far, the company behind 21YunBox 北京二壹科技有限公司 (roughly translates to Beijing 2 1 Technology Co. Ltd.) was founded in 2019 by @tobyglei, so it is a relatively new company, but it does seem that fills a void for web hosting in China.  While there are other options for static website hosting in China, 21YunBox is unique and refreshing with its focus on making things easy for DevOps, somewhat similar to Netlify.

21YunBox is currently using Alibaba Cloud as backend.

21YunBox is also mentioned in the documentation of some other SSGs:

* https://hexo.io/docs/one-command-deployment.html#21YunBox
* https://www.gatsbyjs.com/docs/deploying-to-21yunbox/

So, while 21YunBox may not be very notable yet, it is legit, and it is growing, and seems to offer one-stop services in a very clear and simple manner (whereas I personally easily got confused on Alibaba Cloud and Tencent Cloud websites), and I am supportive of 21YunBox's effort.

That said, @tobyglei, https://hugo.21yunbox.com/ definitely needs some love.  Right now, it seems either too empty or broken...  :wink:

(I think eventually someone should write about Deploying Hugo on Alibaba Cloud and Tencent Cloud if we want to be fair, but that is if there are future user contributions...)

**Comment by bep** [comment]
Thanks.

File: content/en/hosting-and-deployment/hosting-on-21yunbox.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    weight: 10
+weight: 10
+sections_weight: 10
+draft: false
+aliases: []
+toc: true
+---
+
+## Introduction
+
```
Comment (bep) [inline content/en/hosting-and-deployment/hosting-on-21yunbox.md:23]: Can you remove the Introduction section.

File: content/en/hosting-and-deployment/hosting-on-21yunbox.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+That's it! Your site will be live on your 21YunBox URL (which looks like `yoursite.21yunbox.com`) as soon as the build is done.
+
+## Continuous deploys
+
+Now that 21YunBox is connected to your repo, it will automatically build and publish your site any time you push to GitHub.
+
+## 21YunBox CDN and cache invalidation
+
+21YunBox hosts your site on a Chinese, blazing fast CDN which ensures the fastest possible download times for all your users across China.
```
Comment (bep) [inline content/en/hosting-and-deployment/hosting-on-21yunbox.md:62]: Remove the sentence above.

**Review by bep — CHANGES_REQUESTED** [review]

**Comment by tobyglei** [comment]
> @bep wrote:
> 
> > 21YunBox currently gives me 1750 hits when searching on Google (which may be a language thing); do you think this is "notable enough"?
> 
> @tobyglei replied:
> 
> > How about you search "21云盒子" on Google, it's the 21YunBox in Chinese.
> > I would say it's the language thing😉 that why you got limited search results.
> > _(screenshot with 32,000,000 results)_
> 
> But not if the search term is double-quoted like so: `"21云盒子"`, which exclude results like "21", "盒子", and we are left with "about 2,040 results", so, no, unfortunately, it is not a "language thing" in this particular case.
> 
> Baidu does give 198,000 results though, so perhaps it is more notable there.
> 
> From what I can gather thus far, the company behind 21YunBox 北京二壹科技有限公司 (roughly translates to Beijing 2 1 Technology Co. Ltd.) was founded in 2019 by @tobyglei, so it is a relatively new company, but it does seem that fills a void for web hosting in China. While there are other options for static website hosting in China, 21YunBox is unique and refreshing with its focus on making things easy for DevOps, somewhat similar to Netlify.
> 
> 21YunBox is currently using Alibaba Cloud as backend.
> 
> 21YunBox is also mentioned in the documentation of some other SSGs:
> 
> * https://hexo.io/docs/one-command-deployment.html#21YunBox
> * https://www.gatsbyjs.com/docs/deploying-to-21yunbox/
> 
> So, while 21YunBox may not be very notable yet, it is legit, and it is growing, and seems to offer one-stop services in a very clear and simple manner (whereas I personally easily got confused on Alibaba Cloud and Tencent Cloud websites), and I am supportive of 21YunBox's effort.
> 
> That said, @tobyglei, https://hugo.21yunbox.com/ definitely needs some love. Right now, it seems either too empty or broken... 😉
> 
> (I think eventually someone should write about Deploying Hugo on Alibaba Cloud and Tencent Cloud if we want to be fair, but that is if there are future user contributions...)



@anthonyfok thanks for the summary of 21YunBox.

It's a fair summary. Few things I want to add on it would be: 

- We use hybrid clouds and CDNs, it will cover all the major cloud providers in China (Ali Cloud, Tencent, Baidu, AWS China, 
and many others). You might not able to ID it from the landing page or other demos we host online.

- We not only support static site hosting, we also support backend runtimes like Go, Python, Ruby, Java, PHP, and managed databases. Our vision to make it easy to host in China like Netlify and Heroku.

File: content/en/hosting-and-deployment/hosting-on-21yunbox.md (hunk 2 above)
Comment (tobyglei) [reply content/en/hosting-and-deployment/hosting-on-21yunbox.md]: removed.

File: content/en/hosting-and-deployment/hosting-on-21yunbox.md (hunk 1 above)
Comment (tobyglei) [reply content/en/hosting-and-deployment/hosting-on-21yunbox.md]: removed.

**Comment by tobyglei** [comment]
Hey guys, we have more content in English after 5 months of hard work making more hits and online presence in English

It's not perfect but we have a site in English and more posts in English. 
- Landing page: https://launch-in-china.21cloudbox.com
- Tutorials on using 21YunBox: https://launch-in-china.21cloudbox.com/how-to-use-21yunbox.html

@bep, every project gotta start somewhere right? 
Even we were not "notable enough" back then, but we are growing and constantly making improvements, I believe Hugo wasn't an overnight success right? It's cumulative efforts created by the open-source community. I'm the guy bridging any English great techs to China, and making it possible and accessible in China.

**Pull request merged**